### PR TITLE
Build on Linux and making UI not bright blue

### PR DESCRIPTION
**Pull request by cmiceli**
This includes changes which we required in order to facilitate building on Arch Linux, tidying up the UI and making things look a bit more presentable. There is another change to the backend which is what I needed to do to run the app (was causing fatals).

Sorry for > 1 change in the request, not worth pulling apart.

### CODE REVIEW

File: client/src/config.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
 
 
-export const GRAPHQL_ENDPOINT = 'https://ses-availability-api.herokuapp.com/graphql';
+export const GRAPHQL_ENDPOINT = 'http://10.1.0.59:8080/graphql';
```
Comment (sdunster) [inline client/src/config.js]: revert

File: client/package.json [hunk 2]
```diff
@@ -9,7 +9,8 @@
 	"dependencies": {
 		"apollo-client": "^1.9.0",
 		"buffer": "^5.0.7",
-		"graphql-tag": "^2.4.2",
+		"expo": "^22.0.3",
+		"graphql-tag": "^2.5.0",
```
Comment (sdunster) [inline client/package.json]: whats this?

Comment (cmiceli) [reply client/package.json]: needed to build this on linux earlier. can probably fuck it off.

File: client/src/config.js (hunk 1 above)
Comment (cmiceli) [reply client/src/config.js]: fair enough

**Comment by cmiceli** [comment]
screenshot_20171112-212445

**Comment by cmiceli** [comment]
not the best screenshot, but with some content it will be molto bene

**Comment by OSPFNeighbour** [comment]
You guys are hilarious. Chris - Sam will show you UI mock ups as reference
for nav bar. We need to drop a few icons and consolidate some pages.

**Pull request merged**